# New version of the file generated with:
# perl -i.orig -0p -e 's|(^#include <ngx_event.h>).*(typedef struct)|$1\n#include <openssl\/rand\.h>\n\nint RAND_bytes\(unsigned char \*buf, int num\);\nint RAND_pseudo_bytes\(unsigned char \*buf, int num\);\n\n$2|ms' src/event/ngx_event_openssl.c;
--- src/event/ngx_event_openssl.c.orig	2014-09-16 16:23:19.000000000 +0400
+++ src/event/ngx_event_openssl.c	2015-04-06 12:16:58.744021793 +0300
@@ -8,7 +8,10 @@
 #include <ngx_config.h>
 #include <ngx_core.h>
 #include <ngx_event.h>
+#include <openssl/rand.h>
 
+int RAND_bytes(unsigned char *buf, int num);
+int RAND_pseudo_bytes(unsigned char *buf, int num);
 
 typedef struct {
     ngx_uint_t  engine;   /* unsigned  engine:1; */
